### PR TITLE
Addresses issue #7077

### LINKED ISSUE #7077: Should `createCamera` still auto-set itself as the default camera?
### Topic

When debugging https://github.com/processing/p5.js/issues/7071 with @Vishal2002, we noticed a bug in my original bug report's code: I was creating a framebuffer camera in `setup` with `createCamera`, and it was accidentally being set as the main canvas's camera. This was happening because the default behaviour of any `createCamera()` call is not only to return a new camera, but also to basically call `setCamera()` on it too. If you don't want that, you have to manually surround it with `push`/`pop` or an equivalent.

Some thoughts:
1. If you have just one camera, this is probably fine
2. If you switch between multiple cameras, it starts to get a little weird because the last one you create is the default one, unless you do something to prevent that
3. You basically never want this behaviour for framebuffer cameras, since they should only be applied between a framebuffer's `begin`/`end`

There are a few ways we could try to resolve this, from lightest to heaviest touch:
- **Call out the weird cases in the docs.** This is probably not that effective since it's easy to miss even when you know the behaviour.
- **Make framebuffer cameras not auto-apply themselves.** This would resolve point 3 without really being a breaking change, because it would have caused buggy behaviour before if you were creating a framebuffer camera outside of `push`/`pop`. It introduces a bit of inconsistency, but the way you use cameras in framebuffers is already a bit inconsistent because you have to apply a camera within each `begin`/`end` due to it not saving state between draws.
- **Make all cameras not auto-apply themselves.** This would perhaps be the most consistent with other creation methods in p5 (they don't usually have side effects), but would also be a breaking change, so this would maybe have to be for 2.0.

### PR DESCRIPTION
**Pull request by Garima3110**
Resolves #7077 
Removed default camera creation code in `createCamera()` method

 Changes:

 Screenshots of the change:

#### PR Checklist

- [x] `npm run lint` passes
- [ ] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by limzykenneth** [comment]
@Garima3110 @davepagurek Just to mention that the test are expected to fail so if the implementation itself looks good it can be merged.

File: src/webgl/p5.Camera.js [hunk 1]
```diff
@@ -709,14 +709,6 @@ p5.prototype.frustum = function (...args) {
 p5.prototype.createCamera = function () {
   this._assert3d('createCamera');
   const _cam = new p5.Camera(this._renderer);
-
-  // compute default camera settings, then set a default camera
-  _cam._computeCameraDefaultSettings();
-  _cam._setDefaultCamera();
```
Comment (davepagurek) [inline src/webgl/p5.Camera.js:715]: I think we still want these two lines, which set the camera settings to be a perspective camera with the defaultparameters. I believe it's just the line below that needs to be taken out to not have it set itself as the active camera.

**Review by davepagurek — COMMENTED** [review]
Thanks for taking this on!

**Comment by Garima3110** [comment]
Sorry for the silly mistake, I've made the changes please take a look, and let me know if something additional needs to be done.
Thankyou!

**Review by davepagurek — APPROVED** [review]
Sorry for the delay, thanks for making these changes! Looks great now!

**Pull request merged**